docs/content/en/_index.html

```diff
@@ -1,31 +1,34 @@
-+++
-title = "EKS Anywhere"
-linkTitle = "EKS-A"
-
-+++
+---
+title: "EKS Anywhere"
+linkTitle: "EKS-A"
+benefits:
+  - title: Simplify Kubernetes Management
+    image: /images/line-server-rack.svg
+    description: >
+      Amazon EKS Anywhere simplifies the creation and operation of on-premises Kubernetes clusters with default component configurations and cluster lifecycle automation based on open source standards.
+  - title: Streamline Support
+    image: /images/line-container-cloud.svg
+    description: >
+      AWS supports all Amazon EKS Anywhere components including the integrated 3rd-party software, enabling customers to reduce support costs and avoid maintenance of redundant open-source and third-party tools.
+  - title: Consistent and Reliable
+    image: /images/line-toolbox.svg
+    description: >
+      Amazon EKS Anywhere builds on the strengths of Amazon EKS Distro and provides open-source software that’s up-to-date and patched with optional integrations with other AWS services for capabilities such as authentication, observability, and key encryption.
```

Comment:
```suggestion
      Amazon EKS Anywhere builds on the strengths of Amazon EKS Distro and provides open source software that’s up-to-date and patched with optional integrations with other AWS services for capabilities such as authentication, observability, and key encryption.
```
I don't know why, but "open source" is generally not hyphenated, even when talking about open source software.